source/includes/_repository_analysis.html.md

```diff
@@ -258,7 +258,13 @@ Sorted by path in ascending order.
 
 ### Query Parameters
 
-[Paginated](#collection-pagination)
+[Paginated](#collection-pagination), [Filterable](#collection-filtering)
+
+Filters include:
+
+| Name | Description | Required? |
+| ---- | ----------- | --------- |
+| filter[path] | Single path, `$in` clause containing list of categories or `$regex` clause containing regular expressions | No |
```

Comment:
are you sure the `$in` operator works for this field too ?

Reply:
Yes, it works as well 👍